inst/templates/ad_adae.R

```diff
@@ -134,9 +134,9 @@ adae <- adae %>%
 
 # Save output ----
 
-dir <- file.path(getwd(), "tmp")
+dir <- tools::R_user_dir("admiral_templates_data", which = "cache")
```

Comment:
Could you add the comment (`# Change to whichever directory you want to save the dataset in`) we had before?

Reply:
Good idea, I'll add this (guess we should do the same for other admiral packages)

Reply:
Yes, it should be consistent across all admiral packages.